src/Notify/Notify.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
     <TargetFrameworks>netstandard2.0;netcoreapp2.0;net462</TargetFrameworks>
-    <Version>2.9.0</Version>
+    <Version>2.10.0</Version>
```

Comment:
hmmm, i wonder if this is a breaking change, should it be 3.0.0?

Reply:
I think it is, let's lean on the safe side

Reply:
@idavidmcdonald, @leohemsted I've updated the last commit to make the version 3.0.0